app/src/main/java/com/android/unio/ui/event/EventCreation.kt

```diff
@@ -1,20 +1,379 @@
+                  context.getString(R.string.event_creation_selected_image_description),
+              modifier = Modifier.fillMaxSize(),
+              contentScale = ContentScale.Crop)
+        } else {
+          Image(
+              painter = painterResource(id = R.drawable.adec),
+              contentDescription =
+                  context.getString(R.string.event_creation_placeholder_image_description),
+              modifier = Modifier.fillMaxSize(),
+              contentScale = ContentScale.Crop)
+          Text(
+              text = context.getString(R.string.event_creation_image_label),
+              modifier = Modifier.align(Alignment.Center))
+        }
+      }
+}
+
+@Composable
+fun DateAndTimePicker(dateString: String, timeString: String, modifier: Modifier) {
+  var isDatePickerVisible by remember { mutableStateOf(false) }
+  var isTimePickerVisible by remember { mutableStateOf(false) }
+  var selectedDate by remember { mutableStateOf<Long?>(null) }
+  var selectedTime by remember { mutableStateOf<Long?>(null) }
+  val context = LocalContext.current
+
+  Row(
+      modifier.fillMaxWidth(),
+  ) {
+    OutlinedTextField(
+        modifier =
+            Modifier.weight(1f).pointerInput(Unit) {
+              awaitEachGesture {
+                // Modifier.clickable doesn't work for text fields, so we use Modifier.pointerInput
+                // in the Initial pass to observe events before the text field consumes them
+                // in the Main pass.
+                awaitFirstDown(pass = PointerEventPass.Initial)
+                val upEvent = waitForUpOrCancellation(pass = PointerEventPass.Initial)
+                if (upEvent != null) {
+                  isDatePickerVisible = true
+                }
+              }
+            },
+        value = selectedDate?.let { convertMillisToDate(it) } ?: "",
+        readOnly = true,
+        onValueChange = {},
+        trailingIcon = { Icon(Icons.Default.DateRange, contentDescription = "Select date") },
+        placeholder = { Text(context.getString(R.string.event_creation_placeholder_date_input)) },
+        label = { Text(dateString) })
+    Spacer(modifier = Modifier.weight(0.05f))
+    OutlinedTextField(
+        modifier =
+            Modifier.weight(1f).pointerInput(Unit) {
+              awaitEachGesture {
+                // Modifier.clickable doesn't work for text fields, so we use Modifier.pointerInput
+                // in the Initial pass to observe events before the text field consumes them
+                // in the Main pass.
+                awaitFirstDown(pass = PointerEventPass.Initial)
+                val upEvent = waitForUpOrCancellation(pass = PointerEventPass.Initial)
+                if (upEvent != null) {
+                  isTimePickerVisible = true
+                }
+              }
+            },
+        value = selectedTime?.let { convertMillisToTime(it) } ?: "",
+        readOnly = true,
+        onValueChange = {},
```

Comment:
Can this line be removed?

Reply:
No this is sadly boilerplate that has to be kept as it doesn't have a default value